src/udev/udev-builtin-blkid.c

```diff
@@ -244,13 +245,16 @@ static int builtin_blkid(struct udev_device *dev, int argc, char *argv[], bool t
         for (;;) {
                 int option;
 
-                option = getopt_long(argc, argv, "oR", options, NULL);
+                option = getopt_long(argc, argv, "o::R", options, NULL);
                 if (option == -1)
                         break;
 
                 switch (option) {
                 case 'o':
-                        offset = strtoull(optarg, NULL, 0);
+                        if (optarg)
+                                err = safe_atoi64(optarg, &offset);
+                        if (err < 0)
+                                goto out;
```

Comment:
Hmm, I still don't get it ;) It seems that git ate up a part of the commit message (a line starting with \#, I assume). If `--offset/-o` is given without an argument, this option does nothing. So maybe `optional_argument` should be changed `required_argument`?

Reply:
yupp, i agree.